backend/src/routes/api/components/list.ts

```diff
@@ -22,13 +22,19 @@ export const listComponents = async (
 
   // Fetch the enabled config maps
   const enabledCMs = await getEnabledConfigMaps(fastify, applicationDefs);
-  const getCSVForApp = (app: ODHApp) =>
+  const getCSVForApp = (app: ConsoleApplication) =>
     operatorCSVs.find(
-      (operator) => app.spec.csvName && operator.metadata.name.startsWith(app.spec.csvName),
+      (operator) => app.spec.csvName && operator.metadata?.name?.startsWith(app.spec.csvName),
     );
 
   // Get the components associated with the installed KfDefs or operators
   const installedComponents = applicationDefs.reduce((acc, app) => {
+    app.spec.isEnabled = true;
+    acc.push(app);
+
+    // start of code may not be needed
+    // if registration is automatic by virtue of the ConsoleApplication, ConsoleDocuments being installed as part of the KfDef Component
+    // then we don't need to check.
```

Comment:
Where would be be getting the applications that are not enabled?

Reply:
> Please run npm run test:integration-fix to update the snapshot in order to fix the integration checks

Will do.

> The plan is to use the RhodsQuickStart CRD (from red-hat-data-services/odh-manifests#94). I think we should use that as the basis and use RhodsApplication and RhodsDocument

I hadn't seen this in odh-manifests. Thanks. I'll update this PR and change ConsoleApplication and ConsoleDocument to RhodsApplication, RhodsDocument resp.

> Where would be be getting the applications that are not enabled?

The current ODHApps would be moved to their KfDef Component or ODH marketplace operator locations. So we would add the RhodsApplication, RhodsDocument(s) yamls as part of their deployment. Until they are moved to their permanent location, they can be under [install/odh/base](https://github.com/red-hat-data-services/odh-dashboard/tree/master/install/odh/base) 

For example OpenVino Operator which installs the OpenVino Server